include/library/spdm_common_lib.h

```diff
@@ -13,6 +13,29 @@
 #include LIBSPDM_CONFIG
 #endif
 
+#if defined(LIBSPDM_ENABLE_SET_CERTIFICATE_CAP) && \
+    !defined(LIBSPDM_ENABLE_CAPABILITY_SET_CERTIFICATE_CAP)
+#ifdef _MSC_VER
+#pragma message("LIBSPDM_ENABLE_SET_CERTIFICATE_CAP is deprecated. Use " \
+    "LIBSPDM_ENABLE_CAPABILITY_SET_CERTIFICATE_CAP instead. This warning will be removed with " \
+    "the next release.")
+#else
+#warning "LIBSPDM_ENABLE_SET_CERTIFICATE_CAP is deprecated. Use" \
+    "LIBSPDM_ENABLE_CAPABILITY_SET_CERTIFICATE_CAP instead. This warning will be removed with " \
```

Comment:
@ekohandel uncrustify wants me to indent the `#warning` string. However that indentation shows up when the warning is printed, which is ugly. Note that MSVC concatenates the string literal components and ignores the indented white space. Any tips or trick?

Reply:
Actually looks like this has been resolved.